index.html

```diff
@@ -893,9 +893,12 @@
 This is a special case of the more general principle that data should not be used for more
 [=purposes=] than the data's subjects understood it was being collected for.
 
-A service should explain how it uses people's data to protect them and other people, and
-how it might additionally use someone's data if it believes that person has broken the
-rules.
+Services sometimes use people's data in order to protect those or other people.
+This is legitimate as long as the extra risk created by the data collection is
+less than the protection it offers and the extra risk isn't concentrated on
```

Comment:
I agree that the original text is abrupt and your iteration is better. My concern is that the blanket granting of legitimacy is too strong here. Risk isn't a single variable, the trade-offs are often complex. (How do you compare the risk from processing of one extra data point on billions of people versus potential serious harm to one?) Who assesses the risk of data collection? This feels way too open to the classic "it's for your own good" or "it's for the greater good" arguments that we've heard a lot from governments and tech companies.

I don't have an immediate improvement, but is there a way to formulate this that doesn't grant safety teams extrajudicial powers?

Reply:
I think a common way to capture the notion of risk vs. reward is the principle of proportionality. So something like "as long as the risk of data retention is measured, proportional to the benefit to users and the public, and the service is accountable for misuse of user data."